server/utils/nunjucksSetup.ts

```diff
@@ -145,6 +145,18 @@ export function registerNunjucks(app?: express.Express): Environment {
     return address ? address.split(', ').filter(line => line.trim().length > 0) : undefined
   })
 
+  njkEnv.addFilter('formatListAsString', (list?: string[]) => {
+    if (list && list.length > 0) {
+      let val = '['
+      list.forEach(item => {
+        val = val.concat(`'${item}',`)
```

Comment:
return `'[${list.join(,)}]'`

I think this would do all of this

Reply:
I'll give it go, thanks. I did think it was a bit convoluted!

Reply:
Close, but not quite! Not quoting the items within the list i.e. [ 'ROLE_XXX', 'ROLE_YYY' ]. Hence why I think it needs to be a bit more convoluted.